subsys/shell/shell_utils.c

```diff
@@ -12,7 +12,7 @@ extern const struct shell_cmd_entry __shell_root_cmds_end[0];
 
 static inline const struct shell_cmd_entry *shell_root_cmd_get(u32_t id)
 {
-	return &__shell_root_cmds_start[id];
+	return  &__shell_root_cmds_start[id];
```

Comment:
I believe we can benefit from this extra space in the future ;)

Reply:
ok, will remove it. For now. This PR already have enough of improvements :)